src/infra/workload/globalresources/variables.tf

```diff
@@ -89,30 +92,34 @@ variable "cosmosdb_collection_itemratings_max_autoscale_throughputunits" {
   default     = 4000 # 4000 is the minimum possible value
 }
 
-# We use example.com for the very first creation of Front Door when we have no backends yet
 variable "backends_BackendApis" {
   type = list(object({
-    address = string
-    weight  = number
-    enabled = bool
+    address                = string
+    privatelink_service_id = string
+    privatelink_location   = string
+    weight                 = number
+    enabled                = bool
   }))
-  default = [{
-    address = "changeme-api.example.com"
-    weight  = 1
-    enabled = true
-  }]
+  default = []
+  # default = [{
+  #   address                = "changeme-api.example.com"
+  #   privatelink_service_id = "" # Optional. Example: "/subscriptions/111111111-22222/resourceGroups/sample-rg/providers/Microsoft.Network/privateLinkServices/sample-pl"
+  #   privatelink_location   = "" # Optional. Example: "westus2"
+  #   weight                 = 1
+  #   enabled                = true
+  # }]
 }
 
-# We use example.com for the very first creation of Front Door when we have no backends yet
 variable "backends_StaticStorage" {
   type = list(object({
     address = string
     weight  = number
     enabled = bool
   }))
-  default = [{
-    address = "changeme-storage.example.com"
-    weight  = 1
-    enabled = true
-  }]
+  default = []
+  # default = [{
+  #   address = "changeme-storage.example.com"
+  #   weight  = 1
+  #   enabled = true
+  # }]
```

Comment:
Same here, maybe consider replacing it with a comment that this is configured at a later stage in the pipeline.